lib/modules/manager/custom/regex/utils.ts

```diff
@@ -85,9 +85,9 @@ export function createDependency(
 export function regexMatchAll(
   regex: RegExp,
   content: string,
-): RegExpMatchArray[] {
-  const matches: RegExpMatchArray[] = [];
-  let matchResult: RegExpMatchArray | null;
+): RegExpExecArray[] {
```

Comment:
why this change?

Reply:
In `RegExpMatchArray` `index` is optional. And this change needs indices.

Because `RegExp.exec` returns `RegExpExecArray | null`, I thought it's ok to change the return type to `RegExpExecArray[]`